webapp/channels/src/sass/routes/_backstage.scss

```diff
@@ -220,7 +226,7 @@
         text-overflow: ellipsis;
     }
 
-    .item-details__row + .item-details__row {
+    .item-details__row+.item-details__row {
```

Comment:
are we sure this sibbling selector won't affect other parts of the page? correct me if I understand this change correctly:

Before: `.item-details__row .item-details__row` -> would apply the styles to all `.item-details__row` elements inside another `.item-details__row` element.

After: `.item-details__row + .item-details__row` will only apply the styles to `.item-details__row` elemnts that **directly** follow one more `.item-details__row` element.

If the above statement is correct, we need to verify that with this change nothing else is visually styled affected.